README.md

```diff
@@ -32,9 +33,51 @@ You should see `Git LFS initialized` or a similar message. **Important**: if you
 
 Now, you can download the repository:
 
-- `git clone --recursive https://github.com/JoshParnell/ltheory.git ltheory`
+- `git clone https://github.com/Limit-Theory-Redux/ltheory.git`
 
-## Compiling
+## Compiling (Option 1: Precompiled Bin)
+
+You'll need need to download a precompiled bin folder.
+1. Go to https://github.com/Limit-Theory-Redux/ltheory/actions/runs/3620791750
+2. Under "Artifacts" Click on "ltheory" and download.
+3. Navigate to the top level of the repository. (`~/Desktop/ltheory`)
+4. Create a new folder named `bin` if it does not exist.
+5. Extract ltheory.zip into `bin`
+6. Your Bin folder should now look like this:
+
```

Comment:
Might be worth adding a quick sentence after this numbered list to advise the reader that they can skip to `Running a Lua App`

You can make that a link by writing something like `[Running a Lua App](#running-a-lua-app)`